### PR TITLE
Added dockerfile for the driver

### PR DESCRIPTION
**Pull request by urmahp**
* Updated README to reflect transfer to Universal Robots GitHub
* This is related to the discussion in #96

### CODE REVIEW

**Review by urrsk — APPROVED** [review]
Looks good.

File: ur_robot_driver/resources/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Source Code for use in connection with a Universal Robots or UR+ product, provided that you include (i) an
+# appropriate copyright notice (“©  [the year in which you received the Source Code or the Source Code was first
+# published, e.g. “2021”] Universal Robots A/S and/or its licensors”) along with the capitalized section of this notice
+# in all copies of the Source Code. By using the Source Code, you agree to the above terms. For more information,
+# please contact legal@universal-robots.com.
+# -- END LICENSE BLOCK ------------------------------------------------
+
+FROM ros:foxy
+
+ENV COLCON_WS=~/workspace/ros_ws_foxy_ur_driver
```
Comment (gavanderhoorn) [inline ur_robot_driver/resources/Dockerfile:33]: Suggestion: use `ARG` so it doesn't end up in the `env` of the image.

File: ur_robot_driver/resources/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+RUN mkdir -p $COLCON_WS/src
```
Comment (gavanderhoorn) [inline ur_robot_driver/resources/Dockerfile:34]:
```suggestion
WORKDIR $COLCON_WS/src
```

`WORKDIR` is essentially `mkdir -p /some/path && cd /some/path`.

File: ur_robot_driver/resources/Dockerfile (hunk 1 above)
Comment (urrsk) [reply ur_robot_driver/resources/Dockerfile:33]: Good point, and you can change it if you like when building it
```suggestion
ARG COLCON_WS=~/workspace/ros_ws_foxy_ur_driver
```

**Review by urrsk — CHANGES_REQUESTED** [review]

**Review by urrsk — APPROVED** [review]

File: ur_robot_driver/resources/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# please contact legal@universal-robots.com.
+# -- END LICENSE BLOCK ------------------------------------------------
+
+FROM ros:foxy
+
+ARG COLCON_WS=~/workspace/ros_ws_foxy_ur_driver
+RUN mkdir -p $COLCON_WS/src
+RUN cd $COLCON_WS \
+    && git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git src/Universal_Robots_ROS2_Driver \
+    && vcs import src --skip-existing --input src/Universal_Robots_ROS2_Driver/Universal_Robots_ROS2_Driver.repos \
```
Comment (gavanderhoorn) [inline ur_robot_driver/resources/Dockerfile:37]:
```suggestion
WORKDIR $COLCON_WS
RUN git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git src/Universal_Robots_ROS2_Driver \
    && vcs import src --skip-existing --input src/Universal_Robots_ROS2_Driver/Universal_Robots_ROS2_Driver.repos
```

**Review by urrsk — APPROVED** [review]

**Comment by urrsk** [comment]
@gavanderhoorn great suggestions.
Do you have additional inputs?

**Review by urrsk — APPROVED** [review]

**Pull request merged**